### PR TITLE
Cleanup android/openssh tools detection

### PR DESCRIPTION
**Pull request by douglas-raillard-arm**

### CODE REVIEW

File: devlib/utils/android.py [hunk 1]
```diff
@@ -694,12 +689,11 @@ def adb_list_devices(adb_server=None):
 
 
 def get_adb_command(device, command, adb_server=None):
-    _check_env()
     device_string = ""
     if adb_server != None:
         device_string = ' -H {}'.format(adb_server)
     device_string += ' -s {}'.format(device) if device else ''
-    return "adb{} {}".format(device_string, command)
+    return f"{adb}{device_string} {command}"
```
Comment (marcbonnici) [inline devlib/utils/android.py:696]: Conflict to resolve here to re-add `LC_ALL`.

**Review by marcbonnici — COMMENTED** [review]
Apologies for taking so long to get back to this. Aside from the conflict in the rebase, looks good to me.

File: devlib/utils/android.py (hunk 1 above)
Comment (douglas-raillard-arm) [reply devlib/utils/android.py]: This is now done in `_get_adb_parts()`

**Comment by douglas-raillard-arm** [comment]
PR rebased and conflicts solved

**Review by marcbonnici — APPROVED** [review]

**Pull request merged**